playbooks/files/export-dcn.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
```

Comment:
note that I tested this script on both OSP16 and tripleomaster, and this seems to work fine.

Reply:
Perhaps we should reference https://docs.openstack.org/project-deploy-guide/tripleo-docs/latest/deployment/standalone.html#extract-deployment-information-from-the-controller-node somewhere in this file.

Reply:
ok, I'll add it.